libcontainer/configs/validate/validator.go

```diff
@@ -128,6 +128,44 @@ func (v *ConfigValidator) cgroupnamespace(config *configs.Config) error {
 	return nil
 }
 
+// convertSysctlVariableToDotsSeparator can return sysctl variables in dots separator format.
+// The '/' separator is also accepted in place of a '.'.
+// Convert the sysctl variables to dots separator format for validation.
+// More info:
+//   https://man7.org/linux/man-pages/man8/sysctl.8.html
+//   https://man7.org/linux/man-pages/man5/sysctl.d.5.html
+// For example:
+// Input sysctl variable "net/ipv4/conf/eno2.100.rp_filter"
+// will return the converted value "net.ipv4.conf.eno2/100.rp_filter"
+func convertSysctlVariableToDotsSeparator(val string) string {
+	if val == "" {
+		return val
+	}
+	sepFunc := func(r rune) bool {
+		return r == '.' || r == '/'
+	}
+	firsSepIndex := strings.IndexFunc(val, sepFunc)
+	if firsSepIndex != -1 {
+		if val[firsSepIndex] == '.' {
+			return val
+		} else {
+			f := func(r rune) rune {
+				switch r {
+				case '.':
+					return '/'
+				case '/':
+					return '.'
+				default:
+					return r
+				}
```

Comment:
```suggestion
				return r
```

Reply:
default:
					return r

You mean to delete these two lines?
But I think it’s not necessary to delete it, and it’s okay to write it in the switch segment.